src/cartridges/int_adyen_controllers/cartridge/js/pages/checkout/amazon.js

```diff
@@ -4,7 +4,7 @@ if(window.amazonCheckoutSessionId) {
     function handleAmazonResponse(response, component) {
         if (response.fullResponse && response.fullResponse.action) {
             component.handleAction(response.fullResponse.action);
-        } else if (response.resultCode === 'Authorised') {
+        } else if (response.resultCode === window.resultCodeAuthorised) {
```

Comment:
Can you include the constants file instead of putting it into the window object?